share/android/src/main/java/com/capacitorjs/plugins/share/SharePlugin.java

```diff
@@ -89,7 +100,7 @@ public void share(PluginCall call) {
         if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1) {
             PendingIntent pi = PendingIntent.getBroadcast(
                 getContext(),
-                REQUEST_SHARE,
+                123,
```

Comment:
Could you explain a little more about what's going on here?

Reply:
The request code chosen here isn't important for the result behavior. It's helpful if you need to differentiate pending intents, but the result behavior we perform in `activityResult` doesn't check request codes for its purpose. I replaced it with a magic number here which probably isn't ideal as just `123` so I could change it to `0` and maybe put a comment here so it isn't confusing?

Reply:
Thanks!

> so I could change it to `0` and maybe put a comment here so it isn't confusing?

Agreed! Would be helpful for the next person who reads the code.